pkg/config/config.go

```diff
@@ -80,6 +80,11 @@ type ContainerdConfig struct {
 	// related information) to snapshotters. These annotations are required by
 	// stargz snapshotter (https://github.com/containerd/stargz-snapshotter).
 	DisableSnapshotAnnotations bool `toml:"disable_snapshot_annotations" json:"disableSnapshotAnnotations"`
+
+	// DiscardContent is a boolean flag to specify whether to allow GC to clean
+	// layers up from the content store after successfully unpacking these
+	// contents to the snapshotter.
```

Comment:
please add text for discard_content (and disable_snapshot_annotations while at it :-) to https://github.com/containerd/cri/blame/master/docs/config.md#L108

Reply:
please indicate the default is false, layers are not removed from the content store during garbage collection.